File: cryptography/ceasar.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

char * cipher(int k, char * p);

int main(void) {
    int key;
    char *plain;
    const int l = strlen(plain);
    printf("Key: ");
    scanf("%i", &key);
    printf("Plaintext: ");
    scanf("%s", plain);
    printf("Ciphertext: %s", cipher(key, plain));
}

char * cipher(int k, char * p) {
    int l = strlen(p);
    char * cipher;
    for (int i = 0; i < l; i++) {
       cipher[i] = tolower(p[i]) + k;
        if (cipher[i] > 122) {
            cipher[i] -= 26;
        }
    }
    return cipher;
}

File: cryptography/ceasar.py
key = int(input("State the number of places to be shifted: "))
plain = input("Plaintext: ")
l = len(plain)
cipher = [0] * l
for i in range(0, l):
    cipher[i] = chr(ord(plain[i].lower()) + key)
    if ord(cipher[i]) > 122:
        cipher[i] = chr(ord(cipher[i]) - 26)
cipher = "".join(cipher)
print("Ciphertext: ", cipher)